### PR TITLE
Update projects: use the lib relying on the mxgraph npm package

### PR DESCRIPTION
**Pull request by tbouffard**
Reflect lib changes introduced by https://github.com/process-analytics/bpmn-visualization-js/pull/1040
This is now easier to integrate bpmn-visualization: no more mxgraph initialization code nor manual mxgraph load in html page.

Require a new bpmn-visualization npm package publish to be used by anyone.
Tested locally with `npm link` on the in development version of bpmn-visualization.

### CODE REVIEW

File: examples/projects/typescript-vanilla-with-rollup/package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "bpmn-visualization": "0.11.0",
-    "mxgraph": "4.1.0"
+    "bpmn-visualization": "0.11.0"
```
Comment (csouchet) [inline examples/projects/typescript-vanilla-with-rollup/package.json:36]: Note: This doesn't work with the 0.11.0, but the next version (0.12.0).

**Review by csouchet — APPROVED** [review]

File: examples/projects/typescript-vanilla-with-rollup/package.json (hunk 1 above)
Comment (tbouffard) [reply examples/projects/typescript-vanilla-with-rollup/package.json:36]: yes, so I am putting the PR back to draft 😸

Comment (tbouffard) [reply examples/projects/typescript-vanilla-with-rollup/package.json]: Use `0.12.0-0` with 0db33b6

Comment (csouchet) [reply examples/projects/typescript-vanilla-with-rollup/package.json]: All good for me :)

**Pull request merged**